package.json

```diff
@@ -23,8 +24,29 @@
   },
   "homepage": "https://github.com/thsvr/weather-app#readme",
   "devDependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.1",
     "webpack": "^4.41.6",
```

Comment:
The main purpose of Webpack is so that you can split your code into different modules, but you have everything in a single file. You should try to modularize your code.